css/style.css

```diff
@@ -128,7 +128,7 @@ pre code {
 	display: inline-block;
 	padding: 15px;
 	margin: 5px;
-	min-width: 330px;
+	min-width: 50%;
```

Comment:
Make it like 40% or 45% because 50% causes the second card (when showing up, i.e. when there is an pre-release in progress) to show up on second line instead of next to it.